pkg/storagev2/inmemstore/transaction.go

```diff
@@ -9,7 +9,7 @@ import (
 	"fmt"
 	"sync"
 
-	"github.com/ethersphere/bee/pkg/storagev2"
+	storage "github.com/ethersphere/bee/pkg/storagev2"
```

Comment:
This change is unnecessary.

Reply:
So my vet plugin adds this on every file save. Not really sure how to turn it off, but I guess once we change the package name at the end, it should remove it.